frontend/packages/dev-console/src/components/pipelineruns/logs/MultiStreamLogs.scss

```diff
@@ -0,0 +1,30 @@
+.odc-multi-stream-logs {
+  &--fullscreen {
+    .co-toolbar {
+      background-color: var(--pf-global--BackgroundColor--100);
+      padding: var(--pf-global--spacer--sm);
+    }
+  }
+  &__container {
+    &.is-edge {
+      .pf-c-alert {
+        display: none;
+      }
+    }
```

Comment:
What is this for?

Reply:
this was there to fix a bug on Edge. I have kept it from the previous pipeline logs component.
@andrewballantyne can I remove this change?

Reply:
Ah, probably not. I tested it on Edge and I didn't see any issue. If I recall correctly, this was because Edge was just throwing errors like a mofo but still showing logs... we can probably trim this out. I think it's only used once, so maybe even remove the class-name.

Reply:
rmeoved!